src/presets/netlify/utils.ts

```diff
@@ -92,6 +92,13 @@ export async function writeHeaders(nitro: Nitro) {
   await fsp.writeFile(headersPath, contents);
 }
 
+export function getStaticPaths(nitro: Nitro): string[] {
+  const publicAssets = nitro.options.publicAssets.filter(
+    (dir) => dir.fallthrough !== true && dir.baseURL && dir.baseURL !== "/"
+  );
+  return ["/.netlify/*", ...publicAssets.map((dir) => `${dir.baseURL}/*`)];
```

Comment:
@pi0 are these `baseURL`s guaranteed to be prefixed with a leading slash? I see some presets handling that but I'm not sure if it's necessary. (This Netlify config does require it.)

Reply:
it should be normalized but we can use `joinURL` from `ufo` to make sure of it.